docs/QATv2.md

```diff
@@ -0,0 +1,60 @@
+# Quantization Aware Training (QAT)
+
+This document aims to explain Quantization Aware Training framework for MAX7800x series microcontrollers. QAT for MAX7800x consists of four main stages:
+Activation Statistics Collection, Activation Threshold Determination, Scale Adjustments, and Weights Quantization.
+
+## Activation Statistics Collection
+
+To train a quantization-aware model, the first step is to collect activation statistics. The activation statistics are collected by running the model on the training dataset. The training script includes the activation statistics collection step(stat_collect() function). The activation statistics are the histogram of the activations for each layer.
+
+## Activation Threshold Determination
+
+After collecting the activation statistics, the next step is to determine the activation thresholds. To do this, first, an outlier removal step based on z-score is applied to the activation statistics. The default z-score is 8.0, and it can be changed by defining a z-score on the qat policy file. A threshold is then selected to encompass the full activation range, followed by an iterative algorithm [1] that minimizes quantization error by adjusting the threshold to balance range and resolution. Scales are calculated as powers of two, making the scaling-down operation more computationally efficient by defining them as bit shift operations at the edge hardware.
```

Comment:
Could you please consider this modification?

The collected statistics are use to  determine the activation thresholds. To do this, first, an outlier removal step based on z-score is applied to the activation statistics. The default z-score is 8.0, and it can be changed by defining a z-score on the qat policy file. Then, an iterative algorithm [1] that minimizes the quantization error by adjusting the threshold to determine the full activation range. This algorithm finds a balance point in the tradeoff between range and resolution. cales are calculated as powers of two, making the scaling-down operation more computationally efficient by defining them as bit shift operations at the edge hardware.

Reply:
Resolved in the latest commit.